assets/src/edit-story/output/utils/styles.js

```diff
@@ -27,6 +27,13 @@ function isHexColorString(s) {
 function CustomStyles() {
   const safeToFullRatio = PAGE_RATIO / FULLBLEED_RATIO;
   const fullToSafeRatio = 1 / safeToFullRatio;
+  const fullBleedHeight = FULLBLEED_RATIO / 2;
```

Comment:
This name doesn't make sense to me. `FULLBLEED_RATIO` == 9/16, so why is "height" == 9/18?

Reply:
Fixed with the new calculation method.